### DISCUSSION TITLE
[灌水]铃偷偷穿上了丽娜的衣服！！！

### DISCUSSION THREAD

**Discussion by EhuanyingK** [share121/inter-knot#1345] [其他]
今天来录像店的时候，看到了铃偷偷穿上了丽娜的衣服，偷偷拍了下来，实在是太可怕了（吓晕
![669e35471f22b](https://github.com/user-attachments/assets/57138f27-4012-4c5f-916e-b3a33e4fd0cd)

**Comment by ZhuYuan-148** [comment]
这这这是铃店长？
![Screenshot_20240810_151823](https://github.com/user-attachments/assets/afc34938-2475-4c74-b9b4-6fe4eeaa29dc)

**Comment by fu-fu404** [comment]
早就觉得那对录像店兄妹有问题了，他们上个月据说店费是别人的五倍

**Comment by EhuanyingK** [comment]
有这回事？？？

**Comment by gaojiaobei** [comment]
录像店长已经开始穿女仆装招待客人了吗，这年头啥活都不好干啊

**Comment by WORLD-executeME** [comment]
铃店长可爱捏
